### PR TITLE
feat: walrus file for walrus

### PR DESCRIPTION
**Pull request by shanewxy**
**Problem:**

Provide a walrus file for walrus.

**Related Issue:**
#1928

### CODE REVIEW

File: deploy/manifests/walrus-file.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  - name: walrus-casdoor
+    type: containerservice
+    attributes:
+      containers:
+        - image: sealio/casdoor:v1.344.0-seal.1
+          ports:
+            - internal: 8000
+              external: 8000
+          execute:
+            privileged: true
```
Comment (gitlawr) [inline deploy/manifests/walrus-file.yaml:20]: Do we need this?

Comment (shanewxy) [reply deploy/manifests/walrus-file.yaml:20]: Referred to docker-compose.yaml, but removing it seems no difference.

Comment (gitlawr) [reply deploy/manifests/walrus-file.yaml]: Let's also remove it in compose if it's useless

Comment (shanewxy) [reply deploy/manifests/walrus-file.yaml]: removed

**Review by thxCode — APPROVED** [review]
LGTM

**Review by gitlawr — APPROVED** [review]
LGTM

**Pull request merged**